kie-internal/src/main/java/org/kie/internal/task/api/InternalTaskService.java

```diff
@@ -39,32 +38,24 @@
 import org.kie.internal.task.query.TaskSummaryQueryBuilder;
 
 /**
- * The Task Service Entry Point serves as 
+ * The Task Service Entry Point serves as
  *  facade of all the other services, providing a single entry point
  *  to access to all the services
  */
 public interface InternalTaskService extends TaskService {
-    
-    List<TaskSummary> getTasksOwned(String userId, List<Status> status, QueryFilter filter);
-    
-    List<TaskSummary> getTasksAssignedAsPotentialOwner(String userId, List<String> groupIds, List<Status> status, QueryFilter filter);
-    
+
     void addGroup(Group group);
 
     void addUser(User user);
 
     int archiveTasks(List<TaskSummary> tasks);
 
-    void claim(long taskId, String userId, List<String> groupIds);
-
-    void claimNextAvailable(String userId, List<String> groupIds);
```

Comment:
What was the reason to remove these methods? (sorry if they were just moved, but I could not find where...)